### PR TITLE
Made CallbackCommand A Subclass of Command

### PR DESCRIPTION
**Pull request by torch2424**
relates to #123 

This does the correct fix of subclass the Callback Command 😄 

# Example

Screen Shot 2019-10-16 at 12 06 03 PM

### CODE REVIEW

File: packages/wasm-terminal/lib/process/wasi-command.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import WASI from "@wasmer/wasi";
 import WasmFs from "@wasmer/wasmfs";
 
-import { Command, CommandOptions } from "../command-runner/command";
+import Command from "./command";
+import CommandOptions from "./command-options";
```
Comment (syrusakbary) [inline packages/wasm-terminal/lib/process/wasi-command.ts]: Duplicated file?

Comment (torch2424) [reply packages/wasm-terminal/lib/process/wasi-command.ts]: Fixed 😄

**Comment by torch2424** [comment]
@syrusakbary Made requested changes, this is good to go! 😄

File: packages/wasm-terminal/lib/process/process.ts [hunk 2]
```diff
@@ -34,30 +34,26 @@ export default class Process {
...
       );
     } else {
-      this.callbackCommand = commandOptions.callback;
+      this.command = new CallbackCommand(commandOptions);
     }
   }
 
   async start(pipedStdinData?: Uint8Array) {
-    if (this.wasiCommand) {
+    if (this.command instanceof WASICommand) {
```
Comment (syrusakbary) [inline packages/wasm-terminal/lib/process/process.ts]: This should be abstracted by the `Command` class.

The process should not know about if what was provided was a `WASICommand` or a `CallbackCcommand`. Both should work the same way through the same interface.

**Pull request merged**